_posts/eggrolls.md

```diff
@@ -0,0 +1,28 @@
+02-02-2017-egg rolls
+layout: post
+title:  "Egg rolls"
+date:   2017-02-02
+categories: main
+---
+
+![food](http://www.bemindfulbehuman.com/index.php/cambodian-egg-rolls/)
```

Comment:
Hm, this image isn't quite rendering. It's OK to not include a picture if you'd like. 😄